src/pages/Vaults.tsx

```diff
@@ -137,6 +138,29 @@ export function Vaults() {
     error,
   } = useSWR<AxiosResponse, AxiosError>(VAULTS_DASHBOARD_QUERY, subQueryFetcher);
 
+  const vaultDataResponse: VaultsDashboardResponse = vaultsData?.data.data;
+  const [vaultsDataAppended, setVaultsDataAppended] = useState<VaultsDashboardResponse | null>(null);
+
+  const totalVaultsCount = vaultDataResponse?.vaults.totalCount || 1;
+  const pageCount = Math.ceil(totalVaultsCount / 100) - 1;
+  const { data: vaultsNextPages } = useSWR<AxiosResponse, AxiosError>(
+    pageCount ? OPEN_VAULTS_NEXT_PAGES_QUERY(pageCount) : null,
+    subQueryFetcher,
+  );
```

Comment:
Wouldn't it be a wasted network call with `null` if pageCount is falsy?

Reply:
Really? I was checking by passing `null` and it made the network call, resulting in `400`

Reply:
https://swr.vercel.app/docs/conditional-fetching.en-US#conditional

Reply:
That's weird though. Because if you pass `null` network request is made:
![image](https://github.com/agoric-labs/agoric-inter-dashboard/assets/60459172/bd400733-77c5-42eb-b3af-1631774b5459)